dyn/resource_dyn_record_test.go

```diff
@@ -304,3 +328,12 @@ resource "dyn_record" "foobar" {
   type  = "CNAME"
   ttl   = 3600
 }`
+
+const testAccCheckDynRecordConfig_topLevelDomain = `
+resource "dyn_record" "foobar" {
+  zone = "%s"
+  name = ""
```

Comment:
why not make this optional rather than Required?

Reply:
That is a good question 🤔  We probably cloud and should. Let me fix that.

Reply:
now Name's optional - can we remove this line to ensure it works?

Reply:
sure, sorry for being trigger happy :)